### DISCUSSION TITLE
Setup failing with errors

### DISCUSSION THREAD

**Discussion by thulsadume** [Security-Onion-Solutions/securityonion#12794] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16gb

### Storage for /

256gb

### Storage for /nsm

256gb

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Just wondering if someone could point me in the right direction. The initial setup is failing at the end:

[admin@securityonion /]$ sudo cat root/errors.log
[sudo] password for admin:
      Result: False
     Comment: One or more requisite failed: ssl.influxdb_crt
      Result: False
      Result: False
     Comment: One or more requisite failed: ssl.redis_crt
      Result: False
      Result: False
     Comment: One or more requisite failed: ssl.etc_elasticfleet_crt
      Result: False
      Result: False
     Comment: One or more requisite failed: ssl.etc_elasticfleet_logstash_crt
      Result: False
      Result: False
     Comment: One or more requisite failed: ssl.etc_elasticfleet_logstash_key
      Result: False
     Comment: One or more requisite failed: ssl.etc_elasticfleetlumberjack_crt
      Result: False
      Result: False
     Comment: One or more requisite failed: ssl.etc_elasticfleetlumberjack_key

Thanks for any assistance

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Are you able to share your `/root/sosetup.log`?

**Reply by thulsadume** [reply]
Thanks for the response, I wiped that Eval setup and built a Standalone and so far so good.

-Brian
